packages/edit-site/src/components/page-patterns/style.scss

```diff
@@ -1,10 +1,11 @@
 .edit-site-patterns {
+	background: #1e1e1e;
 	border-left: 1px solid $gray-800;
-	background: none;
 	margin: $header-height 0 0;
 	border-radius: 0;
 	padding: 0;
 	overflow-x: auto;
+	min-height: 100%;
```

Comment:
There is something wrong with the styles in patterns page with `no` experiment enabled and you can't scroll. I haven't found the culprit yet..

Reply:
The issue with scroll is larger: I cannot scroll _any_ content area (patterns, templates, pages) with or without the experiment enabled. For example, this is how it works in `trunk`:

https://github.com/WordPress/gutenberg/assets/583546/94c71fb1-6ddf-498d-9671-4b147317218a

Reply:
Ok, I'll check this.